src/controller_examples/zookeeper_controller/exec/zookeeper_lib/helper_funcs.rs

```diff
@@ -37,6 +37,19 @@ pub fn reconcile_zk_node(path: String, uri: String, replicas: String) -> ZKNodeR
     let path_look_up = path_look_up.unwrap();
     match path_look_up {
         Some(_) => {
+            // Update the cluster size
```

Comment:
@ZichengMa Could you add more comments in this branch and the next one (that creates the zk node) to educate the future us that why we need to create and update the zk node? Try to write them as technical documentation, rather than person comments

Reply:
Or write the comment in the exec reconcile_core, which is better I believe